fe-local-treasures/utils/firebase/imageUpload.js

```diff
@@ -0,0 +1,71 @@
+import * as ImagePicker from "expo-image-picker";
+
+export const pickImage = async (setSelectedImage) => {
+  let result = await ImagePicker.launchImageLibraryAsync({
+    mediaTypes: ImagePicker.MediaTypeOptions.All,
+    allowsEditing: true,
+    quality: 1,
+  });
+  if (!result.canceled) {
+    setSelectedImage(result.assets[0].uri);
+  } else {
+    alert("You did not select any image.");
+  }
+};
+
+export const uploadImage = async () => {
```

Comment:
Not sure if this is somewhat duplicating the code from line 50 on the sign up screen